ivy/array/general.py

```diff
@@ -973,13 +973,15 @@ def inplace_increment(
         With :code:`ivy.Array` instance methods:
 
         >>> x = ivy.array([5.7, 4.3, 2.5, 1.9])
-        >>> y = x.inplace_increment(1)
-        >>> print(y)
+        >>> val = ivy.array(1.)
```

Comment:
I've just pushed a fix to master that allows passing a constant directly to `inplace_increment` and `inplace_decrement`, so I'd suggest to revert that to it's previous state. Same applies to the array and container methods for `inplace_increment` and `inplace_decrement`